web_ui/src/pages/camera-page/camera-page.component.tsx

```diff
@@ -35,8 +39,8 @@ export const CameraPage = (): JSX.Element => {
 
     useLowStorage();
 
-    const isPermissionDenied = hasPermissionsDenied(userPermissions);
-    const isPermissionPending = userPermissions === UserCameraPermission.PENDING;
+    const permissionDenied = hasPermissionsDenied(userPermissions);
```

Comment:
I prefer the original name, the new one doesn’t clearly indicate that it’s a boolean.